pkgs/servers/monitoring/grafana/plugins/grafana-oncall-app/default.nix

```
@@ -0,0 +1,13 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin {
+  pname = "grafana-oncall-app";
+  version = "1.5.1";
+  zipHash = "sha256-3mC4TJ9ACM9e3e6UKI5vaDwXuW6RjbsOQFJU5v0wjk8=";
+  meta = with lib; {
+    description = "Developer-friendly incident response for Grafana";
+    license = licenses.agpl3Only;
+    maintainers = lib.teams.fslabs.members;
```

```suggestion
    maintainers = teams.fslabs.members;
```

`lib` is already in the scope due to `with lib;` in line 7.

Probably not worth going through another review cycle, but something I noticed only now.